src/ugeneui/ugeneui.pri

```diff
@@ -18,7 +18,13 @@ LIBS += -L../$$out_dir()
 LIBS += -lU2Core$$D -lU2Designer$$D -lU2Algorithm$$D -lU2Formats$$D -lU2Gui$$D -lU2View$$D -lU2Test$$D -lU2Lang$$D -lU2Private$$D -lbreakpad$$D -lQSpec$$D
 LIBS += $$add_sqlite_lib()
 
-macx: LIBS += -framework Foundation /System/Library/Frameworks/Security.framework/Security
+macx {
+    exists( /System/Library/Frameworks/Security.framework/Security ) {
+        LIBS += -framework Foundation /System/Library/Frameworks/Security.framework/Security
```

Comment:
If and else are the same

Reply:
Are you sure?
```
macx {
    exists( /System/Library/Frameworks/Security.framework/Security ) {
        LIBS += -framework Foundation /System/Library/Frameworks/Security.framework/Security
    } else {
        LIBS += -framework Foundation
    }
}
```

Reply:
It was the same during my comment. May be some github sync issue